.github/workflows/ci.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: master
   pull_request:
-    branches: master
```

Comment:
On most other repos, we have this included:
```yml
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```

Reply:
`[master]` and `master` are equivalent -- though now that I think of it, we really ought to support CI on pull requests against any branch, not just the master. branch.